src/notification.rs

```diff
@@ -290,5 +312,8 @@ mod test {
         check_macro!("workspace/didChangeConfiguration");
         check_macro!("workspace/didChangeWatchedFiles");
         check_macro!("workspace/didChangeWorkspaceFolders");
+
+        // Proposed
+        // check_macro!("window/progress");
```

Comment:
```suggestion
        #[cfg(feature = "supported")]
        check_macro!("window/progress");
```

And try compiling with and without `supported` on CI.